model/project_ref.go

```diff
@@ -2244,55 +2266,57 @@ func (p *ProjectRef) GetGithubProjectConflicts() (GithubProjectConflicts, error)
 	return res, nil
 }
 
-// ValidateProjectCreation returns a boolean if you should surface the error or not.
-func ValidateProjectCreation(projectId string, config *evergreen.Settings, projectRef *ProjectRef) (bool, error) {
+// shouldValidateTotalProjectLimit will return true if:
+// - we are creating a new project
+// - the original project was disabled
+func shouldValidateTotalProjectLimit(isNewProject bool, originalMergedRef *ProjectRef) bool {
+	return isNewProject || !originalMergedRef.IsEnabled()
+}
+
+// shouldValidateOwnerRepoLimit will return true if:
+// - we are creating a new project
+// - the original project was disabled
+// - the owner or repo has changed
+// - the owner/repo is part of exception
```

Comment:
if the owner repo is NOT an exception right? Maybe "will return true if the owner/repo is not exempt from validation and: <list>"

Reply:
yes!